frontend/main.js

```diff
@@ -124,6 +125,16 @@ async function startApp () {
       // setTimeout(() => {
       //   this.$refs.bannerGeneral.show(this.L('Trying to reconnect...'), 'wifi')
       // }, 2500)
+
+      if (this.ephemeral.isCorrupted) {
+        this.$refs.bannerGeneral.danger(
+          this.L('Your app seems to be corrupted. Please {a_}re-sync your app data.{_a}', {
+            'a_': '<a class="link" href="/app/dashboard?modal=UserSettingsModal&section=troubleshooting">',
```

Comment:
So... do we want to be able to access this page while we are not part of a group?

I think we might want to be able to do that... e.g. if the app is corrupted and therefore we can't access our group, it might make sense to be able to visit this as long as we're logged in.

That can be done either in this PR or in a future PR (in which case an issue is needed for that). Your choice!

Reply:
Hum... nice question. I made it work on the last commit. Please take a look.

Reply:
I think the file `router.js` needs to be updated... it just redirects back to welcome page to create a group. You can add a TODO to for that or implement in this PR, your choice!

Reply:
Why do we need to update `router.js` exactly? I tested and it worked - it opens the recovery page.

Note that this is dumb: The URL is created on mount, so if we change pages, the URL is not updated (the pathname), and that's maybe why you are facing problems. This was just a dummy implementation of the banner link. IRL the banner would appear at the right moment, and there the URL would be correct.

Reply:
> Why do we need to update router.js exactly? I tested and it worked - it opens the recovery page.

... weird. Doesn't work for me 🤷‍♂️ 

I copy/pasted the URL and manually loaded it. Was still redirected to the welcome page where it asks you to create a group. I'm guessing it's one of the guards that's causing it. Did you try in Firefox?

Reply:
When you open a page with no group and click the link, it should start with `/app?modal=`. When you open a tab with group, the link should start with `/app/dashboard/modal=`

Reply:
Ah! You're right I must've been copying the wrong link! Will approve now!